tests/unit/src/test/scala/tests/DefinitionLspSuite.scala

```diff
@@ -233,7 +233,7 @@ class DefinitionLspSuite
            |package a
            |object B/*L1*/ {
            |  def main/*L2*/() = {
-           |    println/*Predef.scala*/(A/*;A.scala:2;A.scala:3*/("John"))
```

Comment:
I think the idea here was to show both object and case class. Any idea why this changed?